src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -1678,8 +1678,8 @@ uint32 Unit::CalcSpellResistedDamage(Unit* victim, uint32 damage, SpellSchoolMas
         if (spellInfo->HasAttribute(SPELL_ATTR4_IGNORE_RESISTANCES))
             return 0;
 
-        // Binary spells can't have damage part resisted
-        if (spellInfo->HasAttribute(SPELL_ATTR0_CU_BINARY_SPELL))
+        // Damage dealt by binary spells cast by players cannot be partially resisted
+        if (GetTypeId() == TYPEID_PLAYER && spellInfo->HasAttribute(SPELL_ATTR0_CU_BINARY_SPELL))
```

Comment:
For partial resists on non binary spells ? The problem with binary resists in PVE is that they can have bad effect on the fight. I remember for example on the jormungars at the trial of the crusader, once in a while the tank was resisting the bite mechanic, and therefore, other players could not go to him to get the other debuff removed.
If not, then should IGNORE_HIT_RESULT attribute have the same effect as IGNORE_RESISTANCE for binary spells (if it is not already the case, maybe it is).